reference/entry.json

```diff
@@ -670,6 +670,108 @@
         ]
       }
     },
+    "/events/{eventId}/videos": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "post": {
+        "summary": "Add a video to an event",
+        "operationId": "post-event-video",
+        "responses": {
+          "200": {
+            "description": "The video was added to the event.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "videoId": {
+                      "type": "string",
+                      "format": "uuid",
+                      "example": "b504cf44-9ab8-4641-9934-38d1cc67242c"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request. Possible error types:\n\n* https://api.publiq.be/probs/body/missing\n* https://api.publiq.be/probs/body/invalid-syntax\n* https://api.publiq.be/probs/body/invalid-data",
```

Comment:
Is there an error type specifically for invalid video URLs (not YouTube or Vimeo) that needs to be documented? Or do you return a `https://api.publiq.be/probs/body/invalid-data` error for those cases?

Reply:
This is something that still needs to be implemented in `udb3-silex` based on the JSON schemas. But I would go for a rather generic error `https://api.publiq.be/probs/body/invalid-data`

Reply:
If you use the regex like I mentioned in another comment this will automatically become `invalid-data` yes.